worlds/ff1/Items.py

```diff
@@ -20,7 +20,7 @@ class ItemData(NamedTuple):
 
 FF1_PROGRESSION_LIST = [
     "Rod", "Cube", "Lute", "Key", "Chime", "Oxyale",
-    "Ship", "Canoe", "Floater", "Mark", "Sigil", "Canal",
+    "Ship", "Canoe", "Floater", "MARK", "SIGIL", "Canal",
```

Comment:
capitalization was fixed in current FFR beta, it shouldn't be changed in ap at this point